### PR TITLE
#167594002: refactor create event method

### PR DESCRIPTION
**Pull request by Nta1e**
#### What Does This PR Do?
- Expands the `create_event` method to allow a user to decide whether to add an event to their calendar

#### Description Of Task To Be Completed
- Add a boolean argument to the mutation method for checking whether a user wants to add the event to their calendar or not
#### Any Background Context You Want To Provide?
- None
#### How can this be manually tested?
- Checkout the branch and make the necessary migrations
- make the following mutation to create an event
```
mutation createEvent{
  createEvent(
    input: {
      description: "Just be there",
      title:"Wilding",
      venue: "Central park",
      startDate: "2019-08-02T17:00:19.950122+00",
      endDate: "2019-08-02T17:30:19.950122+00"
    	featuredImage: "whenTheySeeUs.img",
      categoryId: "Q2F0ZWdvcnlOb2RlOjI=",
      timezone: "Africa/Kampala",
      frequency:Monthly,
      recurring:false,
      recurrenceEndDate: "2019-08-23T19:30:19.950122+00",
      addToCalendar: false
  }
  ){
    newEvent{
      id
      title
    }
  }
}
```
Note:
	- Set the `addToCalendar` argument to true if you want the event to be added to your calendar 🙂
#### What are the relevant pivotal tracker stories?
- [#167594002](https://www.pivotaltracker.com/story/show/167594002)
#### Screenshot(s)
- None Required 😬

### CODE REVIEW

File: server/graphql_schemas/event/schema.py [hunk 1]
```diff
@@ -130,6 +132,7 @@ def create_event(category, user_profile, recurrence_event, **input):
     def mutate_and_get_payload(cls, root, info, **input):
         category_id = from_global_id(input.pop('category_id'))[1]
         recurrence_event = None
+        add_to_calendar = input.get('add_to_calendar')
```
Comment (andrewinsoul) [inline server/graphql_schemas/event/schema.py:168]: You can pass in True to be the default value if the add_to_calendar key is not found

File: server/graphql_schemas/event/schema.py [hunk 2]
```diff
@@ -147,7 +150,10 @@ def mutate_and_get_payload(cls, root, info, **input):
                 "recurrence_event": recurrence_event,
                 "user_profile": user_profile
             }
-            cls.start_background_task(**args_dict)
+            if add_to_calendar is None:
```
Comment (andrewinsoul) [inline server/graphql_schemas/event/schema.py:186]: Delete this if-statement

**Review by andrewinsoul — CHANGES_REQUESTED** [review]

File: server/graphql_schemas/event/schema.py (hunk 1 above)
Comment (Nta1e) [reply server/graphql_schemas/event/schema.py:168]: ah! how could I have missed that 🤦🏽‍♂️

**Review by andrewinsoul — APPROVED** [review]
LGTM

**Pull request merged**